src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/RuleValidation/BoosterNotificationRuleValidationError+Equatable.swift

```diff
@@ -0,0 +1,32 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+public enum BoosterNotificationRuleValidationError: Error {
```

Comment:
I would not expect the definition of the error enum in a file with a name implicating an extension implementation. (+Equatable)

ValidationError.swift is the better place. The extension can stay here.